PyPortal_CircuitPython_2020/code.py

```diff
@@ -0,0 +1,84 @@
+"""
+This example will figure out the current local time using the internet, and
+then draw out a countdown clock until an event occurs!
+Once the event is happening, a new graphic is shown
+"""
+import time
+import board
+from adafruit_pyportal import PyPortal
+from adafruit_bitmap_font import bitmap_font
+from adafruit_display_text.label import Label
+
+# The time of the thing!
+EVENT_YEAR = 2020
+EVENT_MONTH = 9
+EVENT_DAY = 9
+EVENT_HOUR = 0
+EVENT_MINUTE = 0
+# we'll make a python-friendly structure
+event_time = time.struct_time((EVENT_YEAR, EVENT_MONTH, EVENT_DAY,
+                               EVENT_HOUR, EVENT_MINUTE, 0,  # we don't track seconds
+                               -1, -1, False))  # we dont know day of week/year or DST
+
+# determine the current working directory
+# needed so we know where to find files
+cwd = ("/"+__file__).rsplit('/', 1)[0]
```

Comment:
You can use `os.getcwd()`, I think.

Reply:
FOr timing sake, ok as is? It matches JP's guides, only code shange was the date for CPD 2020